setup.cfg

```diff
@@ -1,3 +1,52 @@
+[metadata]
+name = fortls
+url = https://github.com/gnikit/fortran-language-server
```

Comment:
urls point to old repo (I can fix that at the very end)

Reply:
Oops, sorry, didn't spot that. I just copied them over from `setup.py`! Will fix